code/go/0chain.net/smartcontract/storagesc/handler.go

```diff
@@ -216,16 +205,88 @@ func (ssc *StorageSmartContract) GetBlobbersHandler(
 
 	var sns storageNodesResponse
 	for _, blobber := range blobbers {
-		sn, err := blobberTableToStorageNode(blobber)
-		if err != nil {
-			return ssc.GetBlobbersHandlerDeprecated(ctx, params, balances)
-		}
+		sn := blobberTableToStorageNode(blobber)
 		sns.Nodes = append(sns.Nodes, sn)
 	}
 	return sns, nil
 }
 
-func (msc *StorageSmartContract) GetTransactionByHashHandler(
+// GetAllocationBlobbersHandler returns list of all blobbers alive (e.g. excluding
+// blobbers with zero capacity).
```

Comment:
Please update comments properly as the function will exclude all blobbers that do not have enough allocation size